android/app/src/main/java/com/expensify/chat/MainActivity.java

```diff
@@ -31,11 +32,11 @@ protected String getMainComponentName() {
    */
   @Override
   protected ReactActivityDelegate createReactActivityDelegate() {
-    return new DefaultReactActivityDelegate(
+    return new ReactActivityDelegateWrapper(this, BuildConfig.IS_NEW_ARCHITECTURE_ENABLED, new DefaultReactActivityDelegate(
```

Comment:
I don't feel strongly or have experience to back this up, my instinct is that departing from the "standard config" for an app including Expo Modules is likely to cause more potential headaches with using upgrades than not.

Reply:
@WojtekBoman or someone else from SWM/Expo- Can you confirm @roryabraham's instincts and/or assumptions above? Thanks.

Reply:
This is the required configuration to use expo in the project :) More information here: https://docs.expo.dev/bare/installing-expo-modules/

Reply:
As far as I know, theoretically it is not required to add those changes, but because that's what the `expo-modules` documentation states I think we should keep it. Especially if we're going to use another module - `expo-av` for video playback.

I think it is also safer to keep those changes, because the expo team would assume that the modules setup is done according to the documentation. It will save us some potential problems during future upgrades.